modules/relocation-of-openshift-jenkins-images.adoc

```diff
@@ -0,0 +1,50 @@
+
+.What stays the same with the OpenShift Jenkins images?
+
+* The Cluster Samples Operator manages the `ImageStream` and `Template` objects for operating the OpenShift Jenkins images.
+* By default, the Jenkins `DeploymentConfig` object from the Jenkins pod template triggers a redeployment when the Jenkins image changes. By default, this image is referenced by the `jenkins:2` image stream tag of Jenkins image stream in the `openshift` namespace in the `ImageStream` YAML file in the Samples Operator payload.
+* If you upgrade from {product-title} 4.10 and earlier to 4.11, the deprecated `maven` and `nodejs` pod templates are still in the default image configuration.
+* If you upgrade from {product-title} 4.10 and earlier to 4.11, the `jenkins-agent-maven` and `jenkins-agent-nodejs` image streams still exist in your cluster. To maintain these image streams, see the "So what happens with the `jenkins-agent-maven` and `jenkins-agent-nodejs` image streams in the `openshift` namespace?" section below.
+
+.What changes in the support matrix of the OpenShift Jenkins image?
+
+Each new image in the `ocp-tools-4` repository in the `registry.redhat.io` registry supports multiple versions of {product-title}. When Red Hat updates one of these new images, it is available for all versions simultaneously. This availability is ideal when Red Hat updates an image in response to a security advisory. Initially, this applies to {product-title} 4.11 and later. Eventually, it is planned that this will apply to {product-title} 4.9 and later.
+
+Previously, each Jenkins image supported only one version of {product-title} and Red Hat might update those images sequentially over time.
+
+.What additions are there with the OpenShift Jenkins and Jenkins Agent Base ImageStream and ImageStreamTag objects?
+
+By moving from an "in-payload" image stream to an image stream that references non-payload images, {product-title} can define as many image stream tags as needed. Red Hat has created a series of new image stream tags to go along with the existing `"value": "jenkins:2"` and `"value": "image-registry.openshift-image-registry.svc:5000/openshift/jenkins-agent-base:latest"` image stream tags present in {product-title} 4.10 and earlier. These new image stream tags address some requests to improve how the Jenkins-related image streams are maintained.
+
+About the new image stream tags:
+
+`ocp-upgrade-redeploy`:: To update your Jenkins image when you upgrade {product-title}, you use this image stream tag in your Jenkins deployment configuration. This image stream tag corresponds to the existing `2` image stream tag of the `jenkins` image stream and the `latest` image stream tag of the `jenkins-agent-base` image stream. It employs an image tag specific to only one SHA or image digest. When changes of note are made to the `ocp-tools-4` image, such as for Jenkins security advisories, Red Hat Engineering updates the Cluster Samples Operator payload.
```

Comment:
See my earlier comments about using an "understood you" So, `use this image stream tag` instead of `you use this image stream tag` 

Either way is fine :)

Reply:
Thanks. I reviewed the content and removed a handful of "understood you's." Good suggestion!

Reply:
Replaced and revised the "of note" phrase, too.